### PR TITLE
Feature/general settings

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: client/app/scripts/liveblog-settings/styles/general-settings.less [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@PADDING: 20px;
+.general-settings {
```
Comment (vied12) [inline client/app/scripts/liveblog-settings/styles/general-settings.less]: It seems similar to `client/app/scripts/liveblog-edit/styles/settings.less`, we should factorize them.

File: client/app/scripts/liveblog-settings/module.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+(function() {
+    'use strict';
+    LiveblogSettingsController.$inject = ['$scope', 'api', '$location', 'notify', 'gettext', '$q'];
+    function LiveblogSettingsController($scope, api, $location, notify, gettext, $q) {
+        //prep the settings
+        $scope.liveblogSettings = {'language': {}, 'theme': {}};
+        $scope.settingsLoading = true;
+        api.languages.query().then(function(data) {
+            $scope.languages = [];
```
Comment (vied12) [inline client/app/scripts/liveblog-settings/module.js]: I don't think you need to instantiate a new array here, You can assign `data._items` directly to `$scope.languages`.

File: client/app/scripts/liveblog-settings/module.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                $scope.languages.push(language);
+            });
+        });
+        api.themes.query().then(function(data) {
+            $scope.themes = [];
+            _.map(data._items, function(theme) {
+                $scope.themes.push(theme);
+            });
+        });
+        $scope.settingsLoading = true;
```
Comment (vied12) [inline client/app/scripts/liveblog-settings/module.js]: This line is already defined line 7.
Actually, if you were using `angular.extend($scope, {})` to define your scope, jsLint would told you that you have a duplicated definition of `settingsLoading`.

File: client/app/scripts/liveblog-settings/styles/general-settings.less (hunk 1 above)
Comment (vladnicoara) [inline client/app/scripts/liveblog-settings/styles/general-settings.less]: I agree, when we'll refactor to get a common service, let's find a way to fix this too

**Pull request merged**